nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-frontend/src/main/nifi/src/app/pages/flow-designer/ui/canvas/items/connection/prioritizers/prioritizers.component.html

```diff
@@ -16,53 +16,64 @@
   -->
 
 <div class="prioritizers flex flex-col gap-y-4" cdkDropListGroup>
-    <div class="flex flex-col">
-        <div>
-            Available Prioritizers
-            <i
-                class="fa fa-info-circle"
-                nifiTooltip
-                [tooltipComponentType]="TextTip"
-                tooltipInputData="Prioritizers that are available to reprioritize FlowFiles in this processors work queue."></i>
-        </div>
-        <div
-            class="prioritizers-list border"
-            cdkDropList
-            cdkDropListSortingDisabled
-            [cdkDropListDisabled]="isDisabled"
-            [cdkDropListData]="availablePrioritizers"
-            (cdkDropListDropped)="dropAvailable($event)">
-            @for (item of availablePrioritizers; track item; let i = $index) {
-                <div
-                    class="prioritizer-draggable-item border-b surface-contrast m-1"
-                    cdkDrag
-                    cdkDragPreviewContainer="parent">
-                    <ng-container
-                        *ngTemplateOutlet="
-                            prioritizerItem;
-                            context: { $implicit: item, i: i, canClose: false }
-                        "></ng-container>
-                </div>
-            }
+    @if (!isDisabled) {
+        <div class="flex flex-col">
+            <div>
+                Available Prioritizers
+                <i
+                    class="fa fa-info-circle"
+                    nifiTooltip
+                    [tooltipComponentType]="TextTip"
+                    tooltipInputData="Prioritizers that are available to reprioritize FlowFiles in this processors work queue."></i>
+            </div>
+            <div
+                class="cdk-list border"
+                cdkDropList
+                cdkDropListSortingDisabled
+                [cdkDropListDisabled]="isDisabled"
+                [cdkDropListData]="availablePrioritizers"
+                (cdkDropListDropped)="dropAvailable($event)">
+                @for (item of availablePrioritizers; track item; let i = $index) {
+                    <div
+                        class="prioritizer-draggable-item border mat-elevation-z2 m-1 font-bold"
+                        cdkDrag
+                        cdkDragPreviewContainer="parent">
+                        <ng-container
+                            *ngTemplateOutlet="
+                                prioritizerItem;
+                                context: { $implicit: item, i: i, canClose: false }
+                            "></ng-container>
+                    </div>
+                }
+            </div>
         </div>
-    </div>
+    }
     <div class="flex flex-col">
         <div>
-            Selected Prioritizers
+            @if (!isDisabled) {
+                Selected Prioritizers
+            } @else {
+                Prioritizers
```

Comment:
When there are no selected prioritizers this is what the user sees. 

![Screenshot 2024-05-03 at 9 31 36 AM](https://github.com/apache/nifi/assets/123395/a2bcf560-4b79-40cb-aaaa-67f2112eea40)

Since we are no longer rendering the source list, when there is nothing selected this looks similar to an `input`. Thoughts on handling this corner case explicitly so avoid this potential confusion?

Reply:
OK updated this UX to display "No value set".